bin/play

```diff
@@ -10,4 +10,4 @@ game = Game.new
 game.welcome_instructions
 game.get_game_type
 game.play_game
-game.outcome
+game.show_outcome
```

Comment:
💯

Reply:
I like the renaming.

_However_, should it be the game launcher's responsibility to run this line?

Reply:
I seems to me that once the launcher has launched the game, it should never be summoned again! :-)